lib/core/realtime/hotelClerk.js

```diff
@@ -24,9 +24,30 @@
 const Bluebird = require('bluebird');
 const { Request } = require('kuzzle-common-objects');
 const kerror = require('../../kerror');
+const debug = require('../../util/debug')('kuzzle:realtime:hotelClerc');
 
 const realtimeError = kerror.wrap('core', 'realtime');
 
+const CHANNEL_ALLOWED_VALUES = ['all', 'in', 'out', 'none'];
```

Comment:
(nitpicking) could be a static getter in the `Channel` class, as this appear to be a dedicated constant to that class.

Reply:
That's what I've done in the first place but eslint/vscode complained.. I will try to bypass them

Reply:
Don't spend time on it, it's just a nitpicking.